### ISSUE TITLE
Invalid CSS in distributed css files

### ISSUE THREAD

**Issue by sdarnell** [vuetifyjs/vuetify#1362]
### Steps to reproduce

I have an app where my main CSS preprocessor is SASS so I include the vuetify min css in one of my sass files:
```
@import "../node_modules/vuetify/dist/vuetify.min";
```
Unfortunately that causes SASS compiler to complain about invalid CSS:
```
Scss compiler error: Error: style declaration must contain a
   value on line 4820 of ../node_modules/vuetify/dist/vuetify.min
   >>   line-height: ;
```
And indeed the css does look to be somewhat odd:
```
subheading {
  color: #424242;
  font-size: 16px;
  font-weight: 400;
  line-height: ;                    <====
  letter-spacing: ;                 <====
  margin-bottom: 16px;
}
```
Similar problems exist in the `body-2`, `body-1`, `caption`, `button`.

### Versions
Vue 2.4.2, Vuetify 0.14.10 (but also seen in current dev branch dist folder)

### What is expected ?
Valid CSS that can be uses as SASS files too.

### What is actually happening ?
Error including CSS in a SASS file.

### Reproduction Link
https://github.com/vuetifyjs/vuetify/blob/5af18ab4c5d1a122a9983d9398923ce2c9b2eae3/dist/vuetify.min.css#L3070

**Comment by johnleider** [comment]
This was fixed, please update your Vuetify version to 0.14.10

**Comment by jacekkarczmarczyk** [comment]
0.14.11*

**Comment by sdarnell** [comment]
Fantastic turn-around. Thanks!

**Comment by johnleider** [comment]
Just to note I actually found the cause of this and fixed it with fd93077c180873dd99fc7958af3bc3c2339346fc
